beacon-chain/core/light-client/lightclient.go

```diff
@@ -243,7 +243,7 @@ func NewLightClientUpdateFromBeaconState(
 	return result, nil
 }
 
-func createDefaultLightClientUpdate(currentSlot primitives.Slot) (interfaces.LightClientUpdate, error) {
+func CreateDefaultLightClientUpdate(currentSlot primitives.Slot) (interfaces.LightClientUpdate, error) {
```

Comment:
Made this function public because it's useful in other places as well. specially in tests.